content/_index.md

```diff
@@ -84,7 +84,7 @@ along the way.
 {{% /row %}}
 
 
-{{% sponsor src="https://kastelo.net/img/logo.svg" href="https://kastelo.net/" alt="Kastelo logo" %}}
```

Comment:
This thing is supposed to be generic for multiple sponsors, so we can't really match on `img.logo` (in the CSS below). Maybe pass a parameter `logo-id` to the shortcode, have that become the id, and refer to that in CSS?